libreyolo/label/boost.py

```
@@ -0,0 +1,246 @@
+    def _run(self, images, labels, accepted_cls, names, started_session, epochs, imgsz, batch):
+        import shutil
+        import tempfile
+
+        tmp = None    # thread-local: each worker owns + cleans only its own temp tree
+        try:
+            import torch
+
+            try:
+                torch.set_num_threads(max(1, (torch.get_num_threads() or 4) - 1))
+            except Exception:  # noqa: BLE001
+                pass
```

Ruff (0.15.15)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)